helm-charts/binderhub/values.yaml

```
@@ -0,0 +1,522 @@
+binderhub:
+  config:
+    BinderHub:
+      use_registry: true
```

```suggestion
      use_registry: true
      build_image: "quay.io/jupyterhub/repo2docker:2022.02.0-30.gc4e01b3"
```

Add repo2docker image tag